src/pages/patientView/copyNumberAlterations/column/CohortColumnFormatter.tsx

```diff
@@ -173,6 +173,24 @@ export default class CohortColumnFormatter {
         }
     }
 
+    public static getDownloadValue(
+        data: DiscreteCopyNumberData[],
+        copyNumberCountCache: CopyNumberCountCache
+    ) {
+        const copyNumberCount = CohortColumnFormatter.getCopyNumberCount(
+            data,
+            copyNumberCountCache
+        )!;
```

Comment:
seems like would be better to allow for undefined in the type rather than use ! since you check for falsy below.

Reply:
@adamabeshouse do you want to update this? Or merge as is?